tools/experiments/experiments.html

```diff
@@ -5,7 +5,13 @@
   <title>AMP Experiments</title>
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <meta name="robots" content="noindex">
-  <script async src="/dist.tools/experiments/experiments.js"></script>
+  <!--
+    TODO(sparhami) Can we make these async? We currently depend on
+    execution order so that the AMP_CONFIG from v0.js does not interfere
+    with the one from the cache-busted v0.js loaded via experiments.js.
+  -->
+  <script defer src="https://cdn.ampproject.org/v0.js"></script>
```

Comment:
Why include the v0.js script here?

Reply:
It is needed to get the `Platform` service, otherwise the `Services.platformFor(win)` returns `null`.